app/controllers/basechurch/v1/bulletins_controller.rb

```diff
@@ -14,4 +30,38 @@ def fetch_sunday_bulletin_id
                          pluck(:id).
                          first
   end
+
+  def encoded_policy
+    Base64.strict_encode64(policy.to_json)
+  end
+
+  def policy
+    {
+      expiration: @expiry,
+      conditions: [
+        { bucket: AwsSettings.bucket },
+        { acl: 'public-read' },
+        { expires: @expiry },
+        { success_action_status: '201' },
+        ['starts-with', '$key', ''],
+        ['starts-with', '$Content-Type', ''],
+        ['starts-with', '$Cache-Control', ''],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.